pom.xml

```diff
@@ -1947,6 +1948,18 @@
                 <version>${spring-boot.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-starter-actuator</artifactId>
+                <version>${spring-boot.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>io.micrometer</groupId>
+                <artifactId>micrometer-registry-prometheus</artifactId>
```

Comment:
this component is rather quick changing! I notice 1.5 is current yet this version isn't that old :-) Dependabot will spot and update, useful to keep it up to date before merging but not critical.

Reply:
Yup, latest version is 1.5. 
Important update for monitoring.

Reply:
Spring Boot Actuator Starter uses micrometer 1.3.6 as a dependency